src/back/project/src/service/service.cpp

```diff
@@ -506,4 +510,8 @@ PagingResult<model::Translation> Service::GetTranslationList(uint32_t start, uin
 	return _repo.Translation().GetList(start, limit);
 }
 
+std::string Service::GetJSONSchemasPath() {
```

Comment:
Значение меняется только при инициализации объекта, поэтому мы можем безопасно вернуть его по ссылке.
```suggestion
const std::string& Service::GetJSONSchemasPath() {
```